crypto/secp256k1/secp256k1.go

```diff
@@ -9,17 +9,18 @@ import (
 	"math/big"
 
 	secp256k1 "github.com/btcsuite/btcd/btcec/v2"
-	"github.com/btcsuite/btcd/btcec/v2/ecdsa"
-	"golang.org/x/crypto/ripemd160" //nolint: staticcheck // necessary for Bitcoin address format
+	ethCrypto "github.com/ethereum/go-ethereum/crypto" //nolint:depguard
```

Comment:
What was the linter complaining about?

Reply:
`import 'github.com/ethereum/go-ethereum/crypto' is not allowed from list 'main' (depguard)`.